tests/acceptance/helpscount-beacon-test.js

```
@@ -0,0 +1,49 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
+import { setupAnimationTest, animationsSettled } from 'ember-animated/test-support';
+import testScenario from 'codecrafters-frontend/mirage/scenarios/test';
+import { waitUntil, visit } from '@ember/test-helpers';
```

ESLint

[error] 'waitUntil' is defined but never used.

(no-unused-vars)